BuildScript.Nuke/Build.Nuget.cs

```diff
@@ -0,0 +1,183 @@
+  private const string c_copyrightPropertyKey = "Copyright";
+
+
+  private Target GenerateNuGetPackagesWithDebugSymbols => _ => _
+      .DependsOn(ImportDefinitions, CompileReleaseBuild, CompileTestBuild)
+      .OnlyWhenStatic(() => !SkipNuGet)
+      .Executes(() =>
+      {
+        ReleaseProjectFiles.ForEach(projectFile =>
+        {
+          GenerateSinglePackageWithDebugSymbols(projectFile);
+        });
+        TestProjectFiles.ForEach(projectFile =>
+        {
+          GenerateSinglePackageWithDebugSymbols(projectFile, c_testFolderSuffix);
+        });
+      });
+
+  private Target GenerateNuGetPackagesWithSymbolServerSupport => _ => _
+      .DependsOn(ImportDefinitions, CompileReleaseBuild, CompileTestBuild)
+      .OnlyWhenStatic(() => !SkipNuGetOrg)
+      .Executes(() =>
+      {
+        ReleaseProjectFiles.ForEach(projectFile =>
+        {
+          GenerateSinglePackageWithSymbolServerSupport(projectFile);
+        });
+        TestProjectFiles.ForEach(projectFile =>
+        {
+          GenerateSinglePackageWithSymbolServerSupport(projectFile, c_testFolderSuffix);
+        });
+      });
+
+  private void GenerateSinglePackageWithDebugSymbols (ProjectMetadata projectFile, string folderSuffix = "")
+  {
+    Configuration.ForEach(
+        config =>
+        {
+          var nugetOutputDirectory = DirectoryHelper.OutputDirectory / c_nugetWithDebugSymbolsFolderName / (config + folderSuffix);
+          FileSystemTasks.EnsureExistingDirectory(nugetOutputDirectory);
+          if (projectFile.IsSdkProject)
+            GeneratePackagesForSdkProjectWithDebugSymbols(projectFile, config, nugetOutputDirectory);
+          else
+            GeneratePackagesForNoneSdkProjectWithDebugSymbols(projectFile, config, nugetOutputDirectory);
+        });
+  }
+
+  private void GenerateSinglePackageWithSymbolServerSupport (
+      ProjectMetadata projectFile,
+      string folderSuffix = "")
+  {
+    Configuration.ForEach(
+        config =>
+        {
+          var nugetOutputDirectory = DirectoryHelper.OutputDirectory / c_nugetWithSymbolServerSupportFolderName / (config + folderSuffix);
+          FileSystemTasks.EnsureExistingDirectory(nugetOutputDirectory);
+          if (projectFile.IsSdkProject)
+            GeneratePackagesForSdkProjectWithSymbolServerSupport(projectFile, config, nugetOutputDirectory);
+          else
+            GeneratePackagesForNoneSdkProjectWithSymbolServerSupport(projectFile, config, nugetOutputDirectory);
+        });
+  }
+
+  private void GeneratePackagesForSdkProjectWithDebugSymbols (ProjectMetadata projectFile, string config, AbsolutePath nugetOutputDirectoryPath)
+  {
+    DotNetTasks.DotNetPack(
+        s => s
+            .SetProject(projectFile.Path)
+            .SetConfiguration(config)
+            .SetVersion(VersionHelper.AssemblyNuGetVersion)
+            .SetOutputDirectory(nugetOutputDirectoryPath)
+            .SetIncludeSource(false)
```

Comment:
`DisableIncludeSource`/`EnableIncludeSource` could be used instead, might be easier to read.